packages/web/components/complex/paginated-table.tsx

```diff
@@ -79,25 +78,31 @@ export const PaginatedTable = ({
         {virtualRows.map((virtualRow) => {
           const row = rows[virtualRow.index] as Row<ObservablePoolWithMetric>;
           return (
-            <Link
```

Comment:
@HarjobandeepSingh why did you remove the `<Link />` component here? In general, we like to use the NextJS components unless there is a reason not to

Reply:
> @HarjobandeepSingh why did you remove the `<Link />` component here? In general, we like to use the NextJS components unless there is a reason not to

open some links in a new tab and others in the same tab. The Link component doesn't directly support this behavior

Reply:
@HarjobandeepSingh couldn't we do something like this work?

```
const handleLinkClick = (e, queryPool) => {
  e.stopPropagation();
  const link = getPoolLink(queryPool);

  if (queryPool.type === "transmuter") {
    window.open(link, '_blank');
  } else {
    router.push(link);
  }
};

// In your JSX
<Link href={getPoolLink(row.original.queryPool)} passHref>
  <a
    style={{
      position: "absolute",
      top: 0,
      left: 0,
      width: "100%",
      height: `${virtualRow.size}px`,
      transform: `translateY(${virtualRow.start - topOffset}px)`
    }}
    onClick={(e) => handleLinkClick(e, row.original.queryPool)}
  >
    {/* Link content */}
  </a>
</Link>
```

Removing the `<Link />` component is a performance regression since NextJS has built in pre-fetching with the `<Link />` component. So let's refactor to use `<Link />`

Reply:
Are you sure about this behaviour? From next 13 "Link" component render a `<a/>` component:
https://nextjs.org/docs/pages/api-reference/components/link#props

https://nextjs.org/docs/pages/api-reference/components/link#legacybehavior

Reply:
Why do you need `passHref` inside `<Link/>`? you can just add style to it (you also avoid one extra nest layer) and also if you use `target=_blank` on `<Link/>` maybe you don't need `onClick` event. Do you get any error if you don't use `onClick` event?

Reply:
its working perfectly without any issues

Reply:
The passHref inside <Link> is crucial for correctly passing the href property to the underlying <a> tag, ensuring proper link functionality. While adding styles directly might be an option, the passHref approach is cleaner and maintains the intended behavior of Next.js navigation. Additionally, using target="_blank" directly on <Link> could open all links in new tabs, affecting global behavior.

Reply:
https://nextjs.org/docs/pages/api-reference/components/link#prefetch

prefetch docs ^^^

Reply:
You can specify `target="_self"` if query type isn't equal to `transmuter`

Reply:
it might not have a significant impact on the results